### PR TITLE
Hotfix/update translations

### PR DESCRIPTION
**Pull request by terotic**
Update translations

### CODE REVIEW

File: src/translations/sv.json [hunk 1]
```diff
@@ -88,7 +88,7 @@
   "legend.person.legend": "Privatpersoner, föreningar och företag kan ansöka om båtplats. Av sökande och företag utanför Helsingfors tas en förhöjd årshyra ut.",
   "legend.overview.title": "Sammanfattning",
   "legend.overview.legend": "Kontrollera att uppgifterna stämmer innan du skickar in ansökan.",
-  "legend.thankyou.title": "Tack, din ansökan har skickats!",
+  "legend.thankyou.title": "Tack, din ansökan har skickats!<br/>Du får snart ett bekräftelsemeddelande till den e-postadress du uppgett.<br/><br/>Om du vill ändra eller annullera din ansökan eller undrar om något som gäller ansökan, skicka e-post till venepaikkavaraukset@hel.fi.",
```
Comment (midori0507) [inline src/translations/sv.json]: `<br />` ?

Comment (terotic) [reply src/translations/sv.json]: Is not allowed?

Comment (midori0507) [reply src/translations/sv.json]: Is it rendered as expect?

Comment (terotic) [reply src/translations/sv.json]: Ok it doesn't, it was used in another instance I thought it works. I revise!

Comment (terotic) [reply src/translations/sv.json]: Done, thanks!

File: src/translations/en.json [hunk 2]
```diff
@@ -15,8 +15,8 @@
...
-  "hero.winter.paragraph.second": "NOTE! Please read the instructions on selecting harbours as well as further details concerning the principles of assigning berths and the processing times for applications on the <a href='{url}'>hel.fi</a> website before completing your application.<br/>If you are interested in storing your boat in the water at your designated boat berth over the winter, please contact us.</p>",
+  "hero.winter.paragraph.first": "Here you can apply for a winter storage space for your boat in the City of Helsinki’s storage locations. Appointed spaces become available only rarely, so please also include locations with no designated spaces in your application.",
+  "hero.winter.paragraph.second": "NOTE! Please read the instructions on selecting storage areas as well as further details concerning the principles of assigning storage spaces and the processing times for applications on the <a href='{url}'>hel.fi</a> website before completing your application.<br/>If you are interested in storing your boat in the water at your designated boat berth over the winter, please contact us.</p>",
```
Comment (midori0507) [inline src/translations/en.json]: Remove `<a>`

Comment (midori0507) [reply src/translations/en.json]: Remove `<p>`

File: src/translations/sv.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
...
   "hero.winter.title": "Ansökan om vinteruppläggningsplatser",
   "hero.winter.paragraph.first": "Här kan du ansöka om vinteruppläggningsplats i något av Helsingfors stads förvaringsområden. Rutor blir sällan lediga så välj också områden utan markerade platser för din ansökan.",
-  "hero.winter.paragraph.second": "OBS! Läs igenom anvisningarna för val av hamnar samt information om såväl reglerna för utdelning av båtplatser som tidsschemat för handläggningen av ansökningar på <a href='{url}'>hel.fi</a> innan du fyller i ansökan.<br/>Kontakta oss om du är intresserad av att lägga upp din båt på din egen båtplats för vintern.</p>",
+  "hero.winter.paragraph.second": "OBS! Läs igenom anvisningarna för val av förvaringsområden samt information om såväl reglerna för utdelning av uppläggningsplatser som tidsschemat för handläggningen av ansökningar på <a href='{url}'>hel.fi</a> innan du fyller i ansökan.<br/>Kontakta oss om du är intresserad av att lägga upp din båt på din egen båtplats för vintern.</p>",
```
Comment (midori0507) [inline src/translations/sv.json]: `br` still here

File: src/translations/en.json (hunk 2 above)
Comment (terotic) [reply src/translations/en.json]: Those are and work as expected

File: src/translations/sv.json (hunk 3 above)
Comment (terotic) [reply src/translations/sv.json]: There it is ok

**Pull request merged**